### PR TITLE
docs: Getting started + React 

### PR DESCRIPTION
**Pull request by toastedtoast**
# Description

Improving the introduction to Liquid Oxygen with a focus on React Developers and advantages in working on projects using Liquid Design. This PR tackles following topics:

- [x] Index
- [x] Why Liquid Oxygen
- [x] Getting Started
- [x] Getting Started React
- [x]  Getting Started Vue (optional)

## Type of change

Please delete options that are not relevant.

- [x] Documentation content changes

## Is it a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -19,19 +19,67 @@ title: Introduction
...
+Web Components are ideal for building reusable, framework-agnostic and future-proof UI components. They are part of the [Web standard](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and are supported by all relevant browsers.
+
+The framework-agnostic nature of Web Components allows you to use Liquid Oxygen in any framework (e.g. React, Vue, Solid or Svelte) or even without a framework.
+
+## Gettings Started
+
+You can add Liquid Oxygen to an existing project. It integrates seamlessly with any framework or library. See our comprehensive guides and detailed explanation of how to integrate Liquid Oxygen into your project.
+
+- [React](https://liquid.emd.design/liquid/introduction/getting-started/react/)
+- [Vue](https://liquid.emd.design/liquid/introduction/getting-started/vue/)
```
Comment (toastedtoast) [inline src/index.md:37]: Remove until Vue example available.

File: src/index.md [hunk 2]
```diff
@@ -19,19 +19,67 @@ title: Introduction
...
+
+Web Components are ideal for building reusable, framework-agnostic and future-proof UI components. They are part of the [Web standard](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and are supported by all relevant browsers.
+
+The framework-agnostic nature of Web Components allows you to use Liquid Oxygen in any framework (e.g. React, Vue, Solid or Svelte) or even without a framework.
+
+## Gettings Started
+
+You can add Liquid Oxygen to an existing project. It integrates seamlessly with any framework or library. See our comprehensive guides and detailed explanation of how to integrate Liquid Oxygen into your project.
+
+- [React](https://liquid.emd.design/liquid/introduction/getting-started/react/)
```
Comment (toastedtoast) [inline src/index.md:36]: Use relative path
```suggestion
- [React](/introduction/getting-started/react/)
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+> Liquid Oxygen works with any package manager (e.g. npm, yarn or pnpm). For simplicity, we continue to use npm for the following guide.
+
+### Import and register web components
+
+The web components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all Liquid Oxygen components. The script loads components lazily only when used in your application. This reduces your bundle size and only loads required code at runtime.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:39]: This is wrong. The cause is confusing terminology from Stencil's side. The components are defined lazily, not loaded lazily. The `defineCustomElements` method is only a convenience method for not defining each component individually. However, it does not have any effect on the bundle size. Nevertheless, the bundler can still tree shake the code, independently of whether the `defineCustomElements` method has been used or not.

```suggestion
The web components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all available Web Components. The script defines the components lazily, only when used in your application. 
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+<!-- > If you plan to use Liquid Oxygen [CSS Components](https://liquid.merck.design/liquid/guides/css-vs-web-components/) you should also refer to the [general Getting Started guide](https://liquid.merck.design/liquid/getting-started/). -->
+
+### Component assets
+
+Some components require static assets during runtime. Although the Liquid Oxygen package includes all assets, you have to add these assets to your output bundle. We recommend to include copying these assets in your build process, which ensures that the assets are always up to date.
+
+> You should add the copied assets (e.g. `public/liquid/assets/*`) to your `.gitignore` file.
+
+For the following example, we assume you are using [vite](https://vitejs.dev/). By default, vite uses the `public` folder for static assets. To include the Liquid Oxygen assets in your output bundle, you can copy them to this folder.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:70]:
```suggestion
For the following example, we assume you are using [Vite](https://vitejs.dev/). By default, Vite uses the `public` folder for static assets. To include the Liquid Oxygen assets in your output bundle, you can copy them to this folder.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+First, install the `rollup-plugin-copy` plugin. This plugin allows you to copy files and folders while building.
+
+```sh
+npm install rollup-plugin-copy -D
+```
+
+Now include the copy plugin in your vite config. Add the following code to your `vite.config.ts` file.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:78]:
```suggestion
Now include the copy plugin in your Vite config. Add the following code to your `vite.config.ts` file.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+npm install @emdgroup-liquid/liquid
+```
+
+> Liquid Oxygen works with any package manager (e.g. npm, yarn or pnpm). For simplicity, we continue to use npm for the following guide.
+
+### Import and register web components
+
+The web components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all Liquid Oxygen components. The script loads components lazily only when used in your application. This reduces your bundle size and only loads required code at runtime.
+
+Add following code to your `App.tsx` file or any similar file which is loaded for every page.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:41]:
```suggestion
Add the following code to your `App.tsx` file (or any similar file which is loaded for every page).
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+// App.tsx
+import { defineCustomElements } from '@emdgroup-liquid/liquid/dist/loader'
+defineCustomElements()
+```
+
+### Import stylesheet
+
+All web components are loaded togehter with their styles embedded. Therefore we only need to import the global stylesheet for Liquid Oxygen.
+
+Add following code to your `App.tsx` file or any similar file which is loaded for every page.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:53]:
```suggestion
Add the following code to your `App.tsx` file (or any similar file which is loaded for every page).
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+```tsx
+// App.tsx
+// if-clause only required in server-side rendering context
+if (typeof window !== 'undefined') {
+  // @ts-ignore: Property '__LD_ASSET_PATH__' does not exist on type 'Window & typeof globalThis'.ts(2339)
+  window.__LD_ASSET_PATH__ = '/liquid/'
+}
+```
+
+Once the asset path is set and the assets are availe on runtime, components can automatically load their assets.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:115]:
```suggestion
Once the asset path is set and the assets are availe on runtime, all components can automatically load their assets.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+If this guide does not fit to your environment, please refer to our sandbox apps for more details and other examples:
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:117]:
```suggestion
If this guide does not suit your environment, please refer to our sandbox apps for more details and other examples:
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+
+- [React & Vite](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-react-tailwind?file=vite.config.ts)<br />This sandbox is quite similar to the example above.
+- [React & CRA](https://codesandbox.io/p/github/emdgroup-liquid/liquid-sandbox-cra-tailwind/main?file=%2Fpackage.json)<br />The Sandbox uses Create React App which does not allow to adjust the webpack config. In this case we added a postinstall script to copy the assets to the public folder.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:120]:
```suggestion
- [React & CRA](https://codesandbox.io/p/github/emdgroup-liquid/liquid-sandbox-cra-tailwind/main?file=%2Fpackage.json)<br />The Sandbox uses Create React App which does not allow to adjust the Webpack config. In this case we added a postinstall script to copy the assets to the public folder.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+- [React & Next.js](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-next-tailwind?file=next.config.js)<br />Next.js uses webpack under the hood. The sandbox shows how to add a custom webpack config `next.config.js` to copy the assets to the public folder.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:121]:
```suggestion
- [React & Next.js](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-next-tailwind?file=next.config.js)<br />Next.js uses Webpack under the hood. The sandbox shows how to add a custom Webpack config `next.config.js` to copy the assets to the public folder.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+
+## Usage
+
+For use in a react project, make sure to take the react bindings of Liquid Oxygen. All components are imported from `@emdgroup-liquid/liquid/dist/react`. The bindings significantly improve JSX compatibility and your developer experience.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:125]:
```suggestion
When adding components from Liquid Oxygen in a React project, make sure you use the React bindings. All components are imported from `@emdgroup-liquid/liquid/dist/react`. The bindings significantly improve `JSX` compatibility and your developer experience.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+    </LdButton>
+  )
+}
+```
+
+When you put this component on a page, you should see a blue button with the text "Click me!" and a lightning bolt icon.
+
+### Events
+
+Liquid Oxygen components aim to stick to the native browser components where possible. In most cases, you can expect the same events and behavior of the Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:146]:
```suggestion
Liquid Oxygen components aim to mimic native HTML elements where possible. In most cases, you can expect the same events and behavior from Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
```

File: src/docs/pages/introduction/getting-started/react.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+    </LdButton>
+  )
+}
+```
+
+Please notice the camel case notation of the `onClick` prop. This is the expected [React convention](https://reactjs.org/docs/handling-events.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and IntelliSense will aid you.
+
+There are a few cases where native events of web components do not behave as expected by React. In these cases, Liquid Oxygen provides custom events prefixed with `ld` and documented on the respective component pages.
+
+> LdInput `onChange` event invokes when the component loses focus (if the value changed), which is the standard browser behavior but differs from the native React `onChange` event. Use the `onInput` event in cases you want to handle user input immediately while typing.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:167]: Maybe we should link to the [event handling guide](https://liquid.merck.design/liquid/guides/event-handling/) here.

Please use the `ld-notice` component instead of the citation markup.

File: src/docs/pages/introduction/getting-started/react.md [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+  return (
+    <LdButton onClick={() => alert('Clicked!')}>
+      Click me!
+      <LdIcon name="energy" />
+    </LdButton>
+  )
+}
+```
+
+Please notice the camel case notation of the `onClick` prop. This is the expected [React convention](https://reactjs.org/docs/handling-events.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and IntelliSense will aid you.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/react.md:163]:
```suggestion
Please notice the camel case notation of the `onClick` prop. This is the expected [React convention](https://reactjs.org/docs/handling-events.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and your code editor's IntelliSense will assist you with that.
```

File: src/index.md (hunk 2 above)
Comment (borisdiakur) [reply src/index.md:36]: I think relative paths should not start with `/` in order to work in all environments, including those with a base path other than `/`.

File: src/docs/pages/introduction/getting-started/index.md [hunk 16]
```diff
@@ -10,9 +10,13 @@ permalink: introduction/getting-started/
 
 # Getting Started
 
+This guide will help you to integrate Liquid Oxygen into your project.
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:13]: Grammarly says works both but I am more used to add the `to` 🤔 
Is this your preference or would you also be fine to keep `to`?

File: src/docs/pages/introduction/getting-started/react.md [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+  parent: Getting started
+  order: 1
+layout: layout.njk
+title: React
+permalink: introduction/getting-started/react/
+---
+
+# React
+
+Liquid Oxygen provides React Bindings for all web components. Although the rendered components are still web components, the bindings improve the developer experience by providing a more familiar API and a better integration with React.
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/react.md:13]: Are you sure about the capital Web Components?

File: src/docs/pages/introduction/getting-started/react.md (hunk 14 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/react.md:167]: Will add a link to the dedicated guide but will keep the notice in here as react devs will probably ignore the link until they run into this issue and get frustrated. As this is basically a pretty common case I'll keep the short version of it in here. + a link to the longer event handler details

File: src/docs/pages/introduction/getting-started/react.md (hunk 12 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/react.md]: Changed it to:
> When adding Liquid Oxygen components to a React project, it is crucial to use the React bindings. All components are imported from `@emdgroup-liquid/liquid/dist/react`. The bindings significantly improve JSX compatibility and your developer experience.

Ok with that?

File: src/docs/pages/introduction/getting-started/react.md (hunk 13 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/react.md]: Changed it to:
> Liquid Oxygen components aim to work similarly to native HTML elements as much as possible. In most cases, you can expect the same events and behavior from a Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.

Ok with that?

File: src/docs/pages/introduction/getting-started/react.md (hunk 12 above)
Comment (borisdiakur) [reply src/docs/pages/introduction/getting-started/react.md]: Yes 👍

File: src/docs/pages/introduction/getting-started/react.md (hunk 13 above)
Comment (borisdiakur) [reply src/docs/pages/introduction/getting-started/react.md]: Yes 👍

File: src/docs/pages/introduction/getting-started/index.md [hunk 18]
```diff
@@ -0,0 +1,221 @@
...
+
+For the CSS Components, you also need to import the global styles and the styles of the components you are using:
+
+```js
+// main.ts
+import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+import '@emdgroup-liquid/liquid/dist/css/ld-button.css'
+```
+
+#### PurgeCSS
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/index.md:90]:
```suggestion
##### PurgeCSS
```

File: src/docs/pages/introduction/getting-started/vue.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+```
+
+## Usage
+
+When adding Liquid Oxygen components to a Vue project, it is crucial to use the Vue bindings. All components are imported from `@emdgroup-liquid/liquid/dist/vue`. The bindings significantly improve Vue compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](components/ld-button/) to your project. This examnple also includes a [LdIcon](components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```html
+// SampleComponent.vue
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/vue.md:59]: This line probably messes up the syntax highlighting. Try this:
```suggestion
<!-- SampleComponent.vue -->
```

File: src/docs/pages/introduction/getting-started/vue.md [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+<script lang="ts">
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+<script>
+
+<template>
+  <LdButton>
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/vue.md:65]: We should use complete examples for Vue here, including the component definition, i.e.:

```html
<script lang="ts">
import { LdTypo } from '@emdgroup-liquid/liquid/dist/vue'
import { defineComponent } from 'vue'

export default defineComponent({
  name: 'App',
  components: {
    LdTypo,
  },
})
</script>

<template>
  <ld-typo>Hello</ld-typo>
</template>
```

Please take note of the `kebab-case` component tags within the template.

File: src/docs/pages/introduction/getting-started/vue.md [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+
+### Events
+
+Liquid Oxygen components aim to work similarly to native HTML elements as much as possible. In most cases, you can expect the same events and behavior from a Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
+
+Let's take our button from above and add a click handler.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/vue.md:82]: This is not quite how a typical component looks like in Vue. You would rather use a [method handler](https://vuejs.org/guide/essentials/event-handling.html#method-handlers). [Here is a nice complete example](https://sfc.vuejs.org/#eNp9UV1LwzAU/SvXILjB1ryXOhQEfVHffMpDs/auzWyTktxWYfS/m2RLEQdCaXK/zrk558QehyGbRmQ5K1xl1UA7ofF7MJagxoMcO4KT0AC1JLlan+8AFmm0OkUAWvaYw93HiNnR3Z2zczjmTfj3SK2pXZ4GGotIK5xQ0wIJwDmU1CpXgtJO1ZjGYDBKkwMyQC1CNVrrB0FWpCYMvSR1hQlFdmhpVb5g1xm4PQXALKw335Tr30yR3VN5XA+qZQR7en+FWEid6gBXeyaOmM9I2gbD0bx5loUivj6JILT/Cr4I7APCfugkYYio2I9ERsND1anq816wKJBgu+dwFvxc9q0FX+bYhqk++LTt5eBVN9p7GHcUl4ITbFFcMG9yiAVriQaXc+4OVXD+6DJjG+5vmR01qR4zdP12b82XQ+uBBYseXjC4T05ot96CGi3a/zD/tF7hJm3Y/AOPytj0).

File: src/docs/pages/introduction/getting-started/react.md (hunk 3 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/react.md]: Think we talked about that last week after your comment:
So based on my observation the mechanics are something like this:

1. default dist: the consuming client registers all components with `defineCustomElements`. Once the browser finds the element in the dom it loads the corresponding script.

2. dist-custom-elements: requires the consuming client to import and register elements individually. This happens when importing the component (due to auto register). This shifts loading the required script for the component to the moment the the component got imported and does not wait (lazily) until it's actually rendered in the dom. 

I think we can omit the details and focus on "recommendations".

File: src/docs/pages/introduction/getting-started/vue.md [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+
+<template>
+  <LdButton @click="say('Clicked!')">
+    Click me!
+    <LdIcon name="energy" />
+  </LdButton>
+</template>
+```
+
+Please notice the @ notation of the `@click` prop. This is the expected [Vue convention](https://vuejs.org/guide/essentials/event-handling.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and your code editor's IntelliSense will assist you with that.
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/vue.md:97]:
```suggestion
Please notice the `@` notation of the `@click` prop. This is the expected [Vue convention](https://vuejs.org/guide/essentials/event-handling.html), which differs from and should not be confused with native [inline event handlers](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#inline_event_handlers_—_dont_use_these). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and your code editor's IntelliSense will assist you with that.
```

File: src/docs/pages/introduction/getting-started/vue.md [hunk 23]
```diff
@@ -0,0 +1,109 @@
...
+
+
+## Sandboxes
+
+This guide shows you how to get started with Liquid Oxygen in your Vue project. Additionally, we provide several sandbox applications showing how to use Liquid Oxygen in various environments:
+
+- [Liquid + Vue + Nuxt](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-nuxt-tailwind)
+- [Liquid + Vue + Vite](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-vue-tailwind)
+
+If you still struggle integrating Liquid Oxygen, feel free to [get in touch with us](https://teams.microsoft.com/l/channel/19%3aeae3b35b0cbf42659e45c2b5592e0c0e%40thread.tacv2/General?groupId=88f23881-53e2-4a99-ad5c-8188c1087bbf&tenantId=db76fb59-a377-4120-bc54-59dead7d39c9).
```
Comment (borisdiakur) [inline src/docs/pages/introduction/getting-started/vue.md:107]: I'm nur sure I'm happy with promoting Teams but not GitHub discussions here. Feels like a closed club to me. I like that we are offering ways to ask for support though. Maybe we can add a get in touch page somewhere at the end of our trouble shooting section, where we list all possibilities to get in touch with us, and link to that page instead?

**Review by borisdiakur — CHANGES_REQUESTED** [review]

File: config/.eleventy.cjs [hunk 24]
```diff
@@ -110,6 +111,15 @@ module.exports = function (eleventyConfig) {
         permalink: renderPermalink,
       })
       .use(markdownItReplaceLink)
+      .use(markdownItLinkAttributes, {
+        matcher(href) {
+          return href.match(/^https?:\/\//)
+        },
+        attrs: {
+          target: '_blank',
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply config/.eleventy.cjs:119]: Added `data-link-external` to outbound links. 
Regarding the icon. I do agree. Will have a look to add a proper icon where applicable.

File: src/docs/pages/guides/component-assets.md [hunk 25]
```diff
@@ -10,14 +10,28 @@ permalink: guides/component-assets/
...
+Some components (e.g. `ld-icon`) require static assets during runtime.
 
-## Tweaking the bundler and adjusting the assets path
+By default, Liquid Oxygen components will fetch these assets via [jsDelivr](https://www.jsdelivr.com/). JsDelivr is a free CDN for open source projects. However, as we cannot guarantee the availability and performance of this CDN, we recommend bundling the assets with your application.
 
-Suppose you are using [Vite](https://vitejs.dev/) which by default requires you to put statically served assets into a folder called [`public`](https://vitejs.dev/guide/assets.html#the-public-directory) in the root directory of your project. You can copy over all Liquid assets into that folder by tweeking the Vite config as follows:
+Copy the assets from the Liquid Oxygen package to your application. We recommend including copying these assets in your build process, which ensures that the assets are always up to date.
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/docs/pages/guides/component-assets.md:17]: Yes, this is good. 👍

File: src/docs/pages/introduction/getting-started/index.md [hunk 26]
```diff
@@ -0,0 +1,221 @@
...
+
+Liquid Oxygen provides a css file containing all styles. Using this file is convenient, but will increase your bundle size significantly. You can use [PurgeCSS](https://purgecss.com/) or something similar, which removes unused CSS classes from your bundle. This is useful, no matter if you are using Web Components or CSS components.
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:92]: I am adding a code block afterwards.

File: src/docs/pages/introduction/getting-started/index.md [hunk 27]
```diff
@@ -0,0 +1,221 @@
...
+title: Getting started
+permalink: introduction/getting-started/
+---
+
+# Getting Started
+
+This guide will help you to integrate Liquid Oxygen into your project.
+
+<ld-notice>
+  If you are using React, please refer to respective <a href="introduction/getting-started/react/">React guide</a>. Similarly, if you are using Vue, please refer to the <a href="introduction/getting-started/vue/">Vue guide</a>.
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:16]: Absolutely right!

File: config/.eleventy.cjs (hunk 24 above)
[the remark this answers is not in the document]
Comment (toastedtoast) [reply config/.eleventy.cjs:119]: Struggle with the icon, but for seeing users there are additional cues to figure that out so for now I omit the icon.
I also added
```
<ld-sr-only id="external-link-description">(external link)</ld-sr-only> 
```

to the template and use this as `aria-describedby` reference. Similar to this guide: 
https://www.digitala11y.com/external-links-in-or-out/

File: src/docs/pages/introduction/getting-started/vue.md (hunk 20 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/vue.md:65]: Based on the stencil documentation defineComponent is not required for lazy dist components which we still unfortunately use because of an issue with tree-shaking of the react / vue output target. https://github.com/ionic-team/stencil-ds-output-targets/issues/255 

Docs for vue & stencil: https://stenciljs.com/docs/vue

although I will adjust all exmples using kebab-case. 
Additionally I add another script tag including setup as this seems to be necessary when importing the components.

File: src/docs/pages/introduction/getting-started/vue.md (hunk 23 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/vue.md:107]: Very good argument.
I'll add the discussions page as the go to place and add teams as an alternative option 🤔. If someone tries to access teams and fails due to be an external user he still may use github discussions. On the other hand I think I'd like to avoid to explain the difference here. Let's  just use the discussions. 

```suggestion
If you still struggle integrating Liquid Oxygen, feel free to [get in touch with us](https://github.com/emdgroup-liquid/liquid/discussions).
```

File: src/docs/pages/introduction/getting-started/index.md [hunk 28]
```diff
@@ -0,0 +1,221 @@
...
+
+Add Liquid Oxygen Web Components to your page just as any other element. You can find detailed documentation of each component in the [components section](components/).
+
+```js
+// main.ts
+import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+
+...
+
+<body>
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:109]: hmmm the vanilla sandbox uses js and imports the css in js. 
There are plenty of ways to do that in vanilla js but I'd try to stick pretty close to the vite template as the whole guide is kind of created around that setup. 

How about having 2 blocks in these cases? one for js one for html?

File: src/docs/pages/introduction/getting-started/index.md [hunk 29]
```diff
@@ -0,0 +1,221 @@
...
+
+Let's take our button from above and add an event listener.
+
+```js
+// main.ts
+<body>
+  <ld-button id="button">Click me</ld-button>
+</body>
+
+document.querySelector <
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:149]: And it's even a syntax error... urrr.

File: src/docs/pages/introduction/getting-started/index.md (hunk 18 above)
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/index.md:90]: hmmm basically a good point. It's not exactly a 3rd option beside web components and css components but it is also not subelement of web components only. 🤔

To be frank, I'd prefer to keep it on this level.

File: src/docs/pages/introduction/getting-started/vue.md (hunk 22 above)
Comment (borisdiakur) [reply src/docs/pages/introduction/getting-started/vue.md]: Historically w3schools has been seen as a flawed resource (see https://meta.stackexchange.com/questions/87678/discouraging-w3schools-as-a-resource). It has improved by now, but it's still a resource which I personally don't trust, like I trust mdn or or the w3c or whatwg docs (which w3schools has no affiliation with, btw). This event attributes page of the w3schools is a perfect example for being out of date.

TLDR: If you just want to list existing event handlers on elements, the document and window object, use this URL:
https://html.spec.whatwg.org/dev/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects

**Review by borisdiakur — CHANGES_REQUESTED** [review]

File: src/docs/pages/introduction/getting-started/vue.md [hunk 30]
```diff
@@ -0,0 +1,118 @@
...
+
+When adding Liquid Oxygen components to a Vue project, it is crucial to use the Vue bindings. All components are imported from `@emdgroup-liquid/liquid/dist/vue`. The bindings significantly improve Vue compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](components/ld-button/) to your project. This examnple also includes a [LdIcon](components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```html
+<!-- SampleComponent.vue -->
+<script setup lang="ts">
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+<script>
```
[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/vue.md:62]: 🤦‍♂️

[the remark this answers is not in the document]
Comment (toastedtoast) [reply src/docs/pages/introduction/getting-started/vue.md:62]: 🤦‍♂️

**Review by borisdiakur — APPROVED** [review]
